### PR TITLE
Update documentation.

### PR DESCRIPTION
**Pull request by Youngestdev**
In this PR, I:

- [x]  Removed How to navigate header.
- [x]  Applied brief explainer to Convoy Cloud & Convoy OSS.
- [x]  Updated the configuration page.
- [x]  Updated the deployment page
- [x]  Updated the user Manual
    - [x]  Changed User Manual to Product Manual.
    - [x]  Added images to Product Manual
- [x]  Added installation procedure to CLI page.

---

I need help with adding the docker documentation page to the sidebar as well as adding the new SDK documentation pages on the sidebar.

### CODE REVIEW

File: content/docs/deploy/docker.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+### Docker
+
+In your terminal, run the command below:
+
+```console[terminal]
+$ docker run \
+	-p 5005:5005 \
+	--name convoy-server \
+	-v `pwd`/convoy.json:/convoy.json \
+	docker.cloudsmith.io/convoy/convoy/frain-dev/convoy:v0.6.5
```
Comment (Subomi) [inline content/docs/deploy/docker.md:28]: Use the `latest` for the docker tag.

File: content/docs/deploy/docker.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+3. Deploy the Convoy application with the command below:
+
+```console[terminal]
+$ docker-compose up
+```
+    
+![Application containers created](/docs-assets/containers-created.png)
+
+![Application running successfully](/docs-assets/healthy-application.png)
```
Comment (Subomi) [inline content/docs/deploy/docker.md:154]: This image doesn't load.

File: content/docs/manual/overview.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: User Manual
+title: Product Manual
```
Comment (Subomi) [inline content/docs/manual/overview.md:2]: This still reads - User Manual both on the sidebar and the page on the deploy preview.

**Review by Subomi — CHANGES_REQUESTED** [review]
@Youngestdev FYI

File: content/docs/overview.md [hunk 4]
```diff
@@ -10,18 +10,18 @@ Convoy
...
----
-### Deploy Convoy
-There are 2 ways of using Convoy: 
-- **Convoy Cloud** - requires no setup, we manage the infrastructure, you publish & receive events. Please note: our cloud platform is in private beta and new users will be accepted on a case-by-case basis.
-- **Self-Host** - requires setup, run in your infrastructure and maintain everything from there. User data stays in your infrastructure.
+We found that APIs are quite pervasive, systems are interconnected as ever. However, there isn’t a defacto technology and tooling for handling all the naunces that comes with publishing, ingesting and debugging webhook events.
+
+Convoy solves that and provides you with options to get started with publishing, ingesting and debugging webhook events.
+
+## Convoy Cloud
```
[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/overview.md:21]: ok.

File: content/docs/manual/overview.md (hunk 3 above)
Comment (Youngestdev) [reply content/docs/manual/overview.md:2]: ok. i'll check what the issue is.

File: content/docs/overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/overview.md:21]: Fixed it to this. Grabbed the definitions from what we currently have:

---

## Convoy Cloud

[Convoy Cloud](https://dashboard.getconvoy.io/signup) provides you a dashboard to manage your webhooks without having to worry about the underlying infrastructure.

## Convoy OSS

Convoy OSS grants you the complete flexibility of setting up Convoy to handle webhooks operations.


Does this work?

[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/overview.md:21]: @Subomi.

[the remark this answers is not in the document]
Comment (Subomi) [reply content/docs/overview.md:21]: Convoy Cloud is our SaaS solution where we manage the underlying infrastructure for you, and you can focus on publishing and receiving webhooks only.

Convoy OSS is our open-source software where you manage the infrastructure by yourselves, and user data never leaves your infrastructure.

[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/overview.md:21]: awesome 👍🏾

File: content/docs/deploy/docker.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+        ports:
+            - 5005:5005
+        depends_on:
+            - mongo
+            - redis_server
+        networks:
+            - backendCluster
+
+    scheduler:
+        image: docker.cloudsmith.io/convoy/convoy/frain-dev/convoy:v0.6.6
```
[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/deploy/docker.md:61]: ok

[the remark this answers is not in the document]
Comment (Youngestdev) [reply content/docs/deploy/docker.md]: fixed.

**Review by Subomi — APPROVED** [review]

**Pull request merged**